src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/tools/jcore/ClassWriter.java

```diff
@@ -601,6 +628,17 @@ protected void writeMethod(Method m) throws IOException {
             dos.writeShort(codeAttrCount);
             if (DEBUG) debugMessage("\tcode attribute count = " + codeAttrCount);
 
+            // write StackMapTable, if available
+            if (hasStackMapTable) {
+              writeIndex(_stackMapTableIndex);
+              dos.writeInt(stackMapAttrLen);
+              // We write bytes directly as stackMapData is
+              // raw data (#entries + entries)
+              for (int i = 0; i < stackMapData.length(); i++) {
+                dos.writeByte(stackMapData.at(i));
+              }
```

Comment:
The indent in this file is 8. So, it has to be 8 in fragments: 528-543 and 632-640.